src/tree.rs

```diff
@@ -494,12 +510,32 @@ impl Tree {
                     res.complete(event);
                 }
 
-                return Ok(Ok(()));
+                return CAS_OK;
```

Comment:
Please remove `CAS_OK` because it's confusing until they look at docs. Everyone knows `Ok(Ok(()))` without having to look at docs, so it's less effort to use.